src/containers/UserEdit/index.jsx

```diff
@@ -362,15 +365,15 @@ class UserEdit extends React.Component {
             onClose={this.handleClose}
           >
             <DialogTitle>
-              {this.state.successMessage || "Password changed successfully!"}
+              {this.state.successMessage || t("password changed successfully")}
             </DialogTitle>
             <DialogActions>
               <Button
                 variant="contained"
                 color="primary"
                 onClick={this.handleClose}
               >
-                OK
+                <Trans>OK</Trans>
```

Comment:
Nit: I believe this can just be 
```suggestion
                <Trans>{t("OK")}</Trans>
```

but lmk if im wrong!

Reply:
@ajohn25 I think the `Trans` element actually just translates the text, so both `Trans` and `t` might be overkill here?

Reply:
ohh i see lol yeah "OK" feels pretty safe to me too